readme.txt

```diff
@@ -42,43 +44,59 @@ Make changes to your site styles, fonts and templates using the Editor.
 
-2. Theme Info editing panel in the WordPress Editor
+- Install and activate a font from any source using the WordPress Font Library.
+- Select "Save Changes To Theme" and select "Save Fonts" to save all of the active fonts to the currently active theme. These fonts will then be activated in the theme and deactivated in the system (and may be safely deleted from the system).
+- Any fonts that are installed in the theme that have been deactivated with the WordPress Font Library will be removed from the theme.
 
-To create a new theme, child theme, or style variation, select "Create Theme" to access the following options for the currently active theme:
+or export the theme:
+
+- Select "Export Zip" to export the theme as a zip file.
+
+To edit the theme metadata:
 
-- "Clone Theme" to create a copy of the active theme.
-- "Create Child Theme" to create a child theme of the active theme.
-- "Create Theme Variation" to create a style variation.
-- "Export Theme" to export the theme as a zip file.
-- "Export Child Theme" to export a child of the currently active theme as a zip file.
-- "Create Blank Theme" to create a new, blank theme with no styles or templates.
+- Select "Edit Theme Metadata" to edit the metadata for the theme. These details will be used in the style.css file.
 
-3. Create Theme panel in the WordPress Editor
+3. Theme Metadata editing panel in the WordPress Editor
 
 To inspect the active theme's theme.json contents:
 
 - Select "Inspect Theme JSON"
 
 4. theme.json inspector in the WordPress Editor
 
-To install and uninstall fonts:
+To create a new blank theme:
 
-- Install and activate a font from any source using the WordPress Font Library.
-- Select "Save Changes" to save all of the active fonts to the currently active theme. These fonts will then be activated in the theme and deactivated in the system (and may be safely deleted from the system).
-- Any fonts that are installed in the theme that have been deactivated with the WordPress Font Library will be removed from the theme.
+- Select "Create Blank Theme"
+- Supply a name for the new theme (and optional additional Metadata)
+- Click "Create Blank Theme"
+
+To create a variation:
+
+- Select "Create Theme Variation"
+- Provide a name for the new Variation
+- Click "Create Theme Variation"
+
+To create a new Clone of the current theme or to create a Child of the current theme:ons for the currently active theme:
+
+- Click "Create Theme"
+- Click "Clone Theme" to create a new Theme based on the active theme with your changes
+- Click "Create Child Theme" to create a new Child Theme with the active theme as a parent with your changes
+
+5. Create Theme panel 1 in the WordPress Editor
+
+6. Create Theme panel 2 in the WordPress Editor
 
 Many of these options are also available under the older, deprecated Create Block Theme page under Appearance > Create Block Theme.
 
-5. Page under Appearance > Create Block Theme
+7. Page under Appearance > Create Block Theme
```

Comment:
Similar here - looks like this is a new screenshot but there's no screenshot-7.jpg file.